examples/07-NBEATS-examples.ipynb

```diff
@@ -37,7 +37,8 @@
     "from darts.models import NBEATSModel\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    from darts.timeseries import concatenate
</pre><p> </p><p>change to</p><pre class="ql-syntax" spellcheck="false">from darts import concatenate
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/2008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>